File: food_order_app/lib/data/requests/register_request.dart
class RegisterRequest {
  String displayName;
  String userName;
  String password;
  String clientId = "Mobile";
  String role;
  String phoneNumber;
  String? avatar;

  RegisterRequest(
      {required this.displayName,
      required this.userName,
      required this.password,
      required this.role,
      required this.phoneNumber,
      this.avatar});

  Map<String, dynamic> toJson() {
    return <String, dynamic>{
      "displayname": displayName,
      "username": userName,
      "password": password,
      "clientId": clientId,
      "role": role,
      "phoneNumber": phoneNumber,
      "avatar": avatar
    };
  }
}

File: FoodOrderApis/AuthServer/Data/Requests/RegisterRequest.cs
namespace AuthServer.Data.Requests;

public class RegisterRequest
{
    public string Displayname { get; set; }
    public string Username { get; set; }
    public string Password { get; set; }
    public string ClientId { get; set; }
    public string Role { get; set; }
    public string PhoneNumber { get; set; }
    public string Avatar {get; set;} = "https://static.vecteezy.com/system/resources/previews/009/292/244/original/default-avatar-icon-of-social-media-user-vector.jpg";
}
